.pipelines/templates/template-podman.yml

```diff
@@ -0,0 +1,163 @@
+
+            if [ -z "$TOKEN" ]; then
+              echo "Error: Failed to retrieve ACR token."
+              exit 1
+            fi
+
+            echo "##vso[task.setvariable variable=ACR_TOKEN]$TOKEN"
+
+      - script: |
+          set -xe
+          export NO_CACHE=true
+          export TOKEN=$(ACR_TOKEN)
+          export REPOSITORY=${{ parameters.repository }}
+          export IMAGE_REPO_NAME=${{ parameters.imageRepoName }}
+
+          # Install podman, make, azure-cli, and Go
+          echo "Installing Podman, Make, Azure CLI, and Go..."
+          sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+          sudo dnf config-manager --add-repo https://packages.microsoft.com/config/rhel/8/prod.repo
+          sudo dnf install -y azure-cli podman make
+
+          # Install Go
+          curl -OL https://golang.org/dl/go1.21.11.linux-amd64.tar.gz
+          sudo tar -C /usr/local -xzf go1.21.11.linux-amd64.tar.gz
+          export PATH=$PATH:/usr/local/go/bin
+
+          # Verify installations
+          which make
+          make --version
+          which podman
+          podman --version
+          which go
+          go version
+          whoami
+
+          # Source the podman control script
+          echo "Sourcing podman control script and starting podman service..."
+          . ./hack/e2e/utils.sh
+          run_podman
+          validate_podman_running
+
+          # Log in to ACR using Podman
+          echo "Logging into ACR with Podman..."
+          echo $TOKEN | podman $PODMAN_REMOTE_ARGS login ${ACR_FQDN} -u "00000000-0000-0000-0000-000000000000" --password-stdin --log-level=debug 2>&1 | tee podman_login.log
+
+          # Check if login was successful
+          if [ $? -ne 0 ]; then
+            echo "Error: Podman login failed."
+            cat podman_login.log
+            kill_podman
+            exit 1
+          fi
+
+          echo "Podman login successful."
+
+          # Run the build and generate a unique image tag
+          echo "Running the build and capturing the image tag..."
+          IMAGE_TAG=$(eval "${{ parameters.makeCommand }}" | grep 'Successfully tagged' | awk '{print $3}' | cut -d':' -f2)
+
+          # If IMAGE_TAG is empty, exit without proceeding to tag and push
+          if [ -z "$IMAGE_TAG" ]; then
+            echo "Warning: No image was built or tagged by the makeCommand. Skipping image push."
+            kill_podman
+            exit 0  # Exit successfully as there's no image to tag or push
+          fi
+
+          # Continue only if an image was successfully built
+          UNIQUE_TAG="${IMAGE_TAG}-$(date +%Y%m%d%H%M%S)-$(Build.BuildId)"
+          podman $PODMAN_REMOTE_ARGS images
+
+          echo "Captured UNIQUE_TAG: $UNIQUE_TAG"
+
+          # Retag the image with the ACR repository and unique tag
+          podman $PODMAN_REMOTE_ARGS tag localhost/${IMAGE_REPO_NAME}:${IMAGE_TAG} ${ACR_FQDN}/${REPOSITORY}:${UNIQUE_TAG}
```

Comment:
idea: replace this by just setting `LOCAL_*_IMAGE` and then make will do that for you

Reply:
Done